client/apps/edit/components/admin/index.jade

```diff
@@ -48,18 +48,18 @@ section#edit-admin.admin-form-container.max-width-container
     .admin-form-right
       .admin-form-right-col
         label Fair
-          #edit-admin-fair
+        #edit-admin-fair
         label Artist Biography
-          #edit-admin-biography
+        #edit-admin-biography
         label Partners
-          #edit-admin-partner
+        #edit-admin-partner
       .admin-form-right-col
         label Editorial Section
-          #edit-admin-section
+        #edit-admin-section
         label Show
-          #edit-admin-shows
+        #edit-admin-shows
         label Auction
-          #edit-admin-auction
+        #edit-admin-auction
```

Comment:
Turns out that autocomplete did not like the that the parent element was a label.